ReportSimulationOutput/tests/test_report_sim_output.rb

```diff
@@ -1296,8 +1328,12 @@ def test_timeseries_energyplus_output_variables
     assert_equal(8760, timeseries_rows.size - 2)
     timeseries_cols = timeseries_rows.transpose
     assert_equal(1, _check_for_constant_timeseries_step(timeseries_cols[0]))
-    _check_for_nonzero_avg_timeseries_value(timeseries_csv, BaseHPXMLTimeseriesColsEnergyPlusOutputVariables)
+    nonzero_avg_cols = BaseHPXMLTimeseriesColsEnergyPlusOutputVariables
+    nonzero_avg_cols -= ['Schedule Value: Always Off Discrete']
+    nonzero_avg_cols -= ['Schedule Value: Central Ac And Furnace Auto Fan Schedule']
+    _check_for_nonzero_avg_timeseries_value(timeseries_csv, nonzero_avg_cols)
     assert(File.readlines(run_log).any? { |line| line.include?("Request for output variable 'Foo'") })
+    _check_for_timeseries_sum(timeseries_csv, 8183.705465, 'Schedule Value: Fridge Schedule')
```

Comment:
I don't really like this test and all the complexity added to test_report_sim_output.rb. It also seems like it will fail in the future based on totally unrelated features that cause the simulation indoor temperatures to change.

Can we just do a simple assert for, say, the ratio of actual energy use to rated energy use? And use a reasonably high tolerance (10%?) so that it won't fail in the future.